docs/covid-19/README.md

```diff
@@ -0,0 +1,96 @@
+
+> “I would shudder to use IgM and IgG testing to figure out who’s immune and who’s not,” says Wells. “These tests are not ready for that.”
+
+Based on [this medicinenet.com article](https://www.medicinenet.com/how_do_the_covid-19_coronavirus_tests_work/article.htm)
+
+> Immunoglobulin tests for COVID-19 cannot confirm the presence of the virus in your system. It can only tell whether you have been exposed in the past or if you have never been exposed to SARS-CoV-2. Consequently, it should only be used alone as a screening test and should be used in tandem with a genetic-based test to determine a complete status. Genetic testing is the gold standard for COVID-19 diagnosis.
+
+### We have to start somewhere...
+
+Lets consider the 4 cases for the "immunoglobulin detection-based tests for COVID-19".
+
+- Negative Result
+- Positive Result, IgM only
+- Positive Result, IgG only
+- Positive Result, IgG and IgM
+
+We only need 2 booleans to represent these cases, one for IgM and one for IgG.
+
+```json
+    ...
+    "IgM": false,
+    "IgG": true,
+```
+
+The rest of this credential is based on https://digitalbazaar.github.io/citizenship-vocab
+
+Uses the new schema.org definitions for COVID-19... see [schema.org](https://schema.org/docs/search_results.html?q=COVID-19).
+
+Photo, name, gender, birthdate useful for matching with existing records, not everyone has a social security number or drivers license...
+
+Embedding image data precludes transmission via QR Codes, worth considering alternative solutions...
+
+This is what a verifier would see:
+
+```json
+{
+  "@context": ["https://www.w3.org/2018/credentials/v1"],
+  "type": ["VerifiablePresentation"],
+  "verifiableCredential": [
+    {
+      "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3c-ccg.github.io/vc-examples/covid-19/v1.jsonld"
+      ],
+      "id": "http://example.com/credential/123",
+      "type": ["VerifiableCredential", "ImmunoglobulinDetectionTestCard"],
+      "issuer": {
+        "id": "did:web:vc.transmute.world",
+        "location": {
+          "@type": "CovidTestingFacility",
+          "name": "Stanford Health Care",
+          "url": "https://stanfordhealthcare.org/"
+        }
+      },
+      "issuanceDate": "2019-12-11T03:50:55Z",
+      "expirationDate": "2020-12-11T03:50:55Z",
+      "name": "Immunoglobulin Detection Test Card",
+      "description": "Immunoglobulin detection tests are based on the qualitative detection of IgM and IgG that are specifically generated by the body in response to SARS-CoV-2 infection.",
+      "credentialSubject": {
+        "id": "did:key:z6MkjRagNiMu91DduvCvgEsqLZDVzrJzFrwahc4tXLt9DoHd",
+        "type": "ImmunoglobulinDetectionTestSubject",
+        "givenName": "Louis",
+        "familyName": "Pasteur",
+        "birthDate": "1958-07-17",
+        "IgM": false,
```

Comment:
JFYI, from a modeling perspective, this doesn't make much sense. Louis has an attribute "IgM" and it's "false"? What does that mean? If it helps, imagine that Louis had lots of VCs for different types of tests like these and he'd like to merge them together to understand his antibody profile. That would be unworkable here.

I would expect some tweaks here to be necessary to instead express that Louis has SARS-CoV-2 antibodies (which can still be broken down by type). I'm not a subject matter expert here, but if an IgM and an IgG antibody are different, then those are different things that a person can have and they should have their own IDs -- that people could hopefully dereference and find out more info about?

```js
"credentialSubject": {
  "id": "did:key:z6MkjRagNiMu91DduvCvgEsqLZDVzrJzFrwahc4tXLt9DoHd",
  "givenName": "Louis",
  // other attributes above, etc. ... and
  "antibody": [
    "<perhaps a specific ID for the SARS-CoV-2 IgM antibody>", // or not present if not a match
    "<perhaps a specific ID for the SARS-CoV-2 IgG antibody>"
  ]
}
```

Anyway, just some late data modeling feedback, take it or leave it.

Reply:
@dlongley thanks for the review! I'm pulling this comment into the v2 thread here: https://github.com/w3c-ccg/vc-examples/issues/34